src/types/match.ts

```diff
@@ -18,6 +18,7 @@ export const chatMessagePayloadSchema = z.object({
   message: z.string(),
   sentAt: z.number(), // unix time
 });
+
```

Comment:
Personally I like the zod types to be attached to their types for clarity so I don't like adding blank lines in between them to make it evident that they are related